multimedia/motion/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=motion
-PKG_VERSION:=4.3.2
+PKG_VERSION:=4.5.0
```

Comment:
Please split this change to separate commit.

Reply:
thanks for the hint. updated PR.